### DISCUSSION TITLE
Doesn't support -DLIB_SUFFIX=64 [5678]

### DISCUSSION THREAD

**Discussion by mtheall** [eProsima/Fast-DDS#3568] [Ideas]
Using the CMake option `-DLIB_SUFFIX=64` installs libraries to `/usr/lib` instead of `/usr/lib64`

**Comment by JLBuenoLopez** [comment]
Moving this to the proper discussion forum according to Fast DDS contributing guidelines.

**Reply by JLBuenoLopez** [reply]
This enhancement is implemented in #1881
